marathon.json

```diff
@@ -0,0 +1,38 @@
+{
+  "id": "hdfs-mesos",
+  "container": {
+    "type": "DOCKER",
+    "docker": {
+      "image": "mesosphere/hdfs-mesos:latest",
+      "network": "HOST"
+    }
+  },
+  "cpus": 0.1,
+  "mem": 256.0,
+  "instances": 1,
+  "env": {
+    "MESOS_HDFS_DATA_DIR": "/tmp/hdfs/data",
```

Comment:
This file is intended to be an example. I just have them in here to show how it's done.